specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,447 @@
+
+## Summary
+
+Soulbound Token (SBT) is a form of a NFT which represents an aspect of an account: _soul_. Transferability is limited only to a case of recoverability or transferring the whole _soul_ - a `sbt_soul_transfer` should coordinate with a registry to transfer all SBTs from one account to another, and _killing_ the source account.
+
+SBTs are well suited for carrying proof-of-attendance, proof-of-unique-human "stamps" and other similar credibility-carriers.
+
+## Motivation
+
+Recent [Decentralized Society](https://www.bankless.com/decentralized-society-desoc-explained) trends open a new area of Web3 research to model various aspects of what characterizes humans. Economic and governance value is generated by humans and their relationship.
+
+> Decentralized Society, or DeSoc, would lean on SBTs to allow “Souls and communities to come together bottom-up, as emergent properties of each other to co-create plural network goods and intelligences, at a range of scales.”
+
+Creating a strong primitives is necessary to model new innovative systems and decentralized societies. Examples include reputation protocols, undercollaterized lending, one-person-one-vote, fair airdrops & ICOs, universal basic income, non KYC identity systems, Human DAOs and methods for Sybil attack resistance.
+
+We propose a SBT standard to model protocols described above.
+
+_Verifiable Credentials_ (VC) could be seen as subset of SBT. However there is an important distinction: VC require set of claims and privacy protocols. It would make more sense to model VC with relation to W3 DID standard. SBT is different, it doesn't require a [resolver](https://www.w3.org/TR/did-core/#dfn-did-resolvers) nor [method](https://www.w3.org/TR/did-core/#dfn-did-methods) registry. For SBT, we need something more elastic than VC.
+
+## Specification
+
+Main requirement for Soulbound tokens is to make it bound to a human. Moving tokens from one account to another should be strictly limited to case of **recoverability** (eg in case a user's private key is compromised due to extortion, loss, etc) or user account merge (**soul transfer**). This becomes especially important for proof-of-human stamps that can only be issued once per user.
+
+Two safeguards against misuse of recovery are contemplated.
+
+1. Users cannot recover an SBT by themselves. The issuer, a DAO or a smart contract (eg: multisig) dedicated to manage the recovery should be assigned.
+2. Whenever a _soul transfer_ is triggered then the SBT registry emits `Kill` event. It creates an inherit cost for such action: the account identity is burned and can't receive any SBT in the future.
+3. The recovery function is additional economical risk preventing account trading: user should always be able to recover his SBT, and move to another, not killed account.
+
+SBT recover MUST not trigger `SoulTransfer` nor `Kill`: malicious issuer could compromise the system by faking the token recovery and take over all other SBTs from a user.
```

Comment:
The meaning or `SoulTransfer` and `Kills` is not clear, it's not described and just suddenly appears from nowhere. There is also a bunch of terms which randomly appears here and there. I suggest to put a basic explanation of the domain at the beginning of the specification, like, "user is ...., soul is..., and there are basic operations: soul kill, soul transfer", and put constraint at the end, like, "alright, we explained everything, now there are some tiny-gritty details"